src/gui/editorwidgets/qgsrelationreferencewidget.cpp

```diff
@@ -208,7 +208,6 @@ void QgsRelationReferenceWidget::setRelation( const QgsRelation &relation, bool
       mComboBox->setIdentifierField( mReferencedField );
 
     mReferencedFieldIdx = mReferencedLayer->fields().lookupField( relation.fieldPairs().at( 0 ).second );
-    mReferencingFieldIdx = mReferencingLayer->fields().lookupField( relation.fieldPairs().at( 0 ).first );
```

Comment:
This should still be initialized I guess?

Reply:
actually, the variable has no use anymore. I just remove it 4ee2a53